log/internal_logger.go

```diff
@@ -0,0 +1,54 @@
+package log
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"net/http"
+)
+
+var analyticsServerURL = "https://bitrise-step-analytics.herokuapp.com"
+
+// Message represents a line in a log
+type Message struct{
+	LogLevel string `json:"log_level"`
+	Message string `json:"message"`
+	Data map[string]interface{} `json:"data"`
+}
+
+func init() {
+	if url := os.Getenv("ANALYTICS_SERVER_URL"); url != "" {
+		analyticsServerURL = url
+	}
+}
+
+// SendToInternal sends the log message to the configured analytics server
+func (lm Message) SendToInternal(stepID, tag string, data map[string]interface{}) {
```

Comment:
`SendToInternal` for me makes no sense. The original `Internal` in the task description would fit better, but that is not a very common name for this type of method. I recommend to find a different name for this.
For example I have a T<Log> and I want that to be saved in our logging tool as well. I would try something like:
- T<Log>.Record()
- T<Log>.Store()
- T<Log>.Send()
- T<Log>.....??()

Reply:
Track I think should be fine, but please check other logging service what they name their method to store a log

Reply:
I don't see why `Send` and `Internal` would be fine, but `SendToInternal` make no sense.

I'll rename it to `Record`.